README.md

```diff
@@ -720,23 +720,30 @@ interface Table extends Parent {
 ### CustomCodeComponent
 
 ```ts
-interface CustomCodeComponent extends Parent {
+interface CccFallbackText extends Node {
```

Comment:
Thought: Why does this need to be specific for CCC? Couldn't be just...
```suggestion
interface FallbackText extends Node {
```
and perhaps in the future be used in other context too?